examples/next-app-directory/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "builder-demo",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@builder.io/react": "^2.0.15",
+    "@builder.io/sdk": "^1.1.33",
```

Comment:
`@builder.io/sdk ` don't need to be a dependency here, it's already included in `@builder.io/react`, it caused an issue for me today testing the remix starter and ended up removing it to get the latest updates

Reply:
https://github.com/BuilderIO/builder/commit/0997d07ad1cfe366b5ff3442149efde697ddb43a#diff-485037290c6f6b4181e1aae71bd2d5da9c25254675a04753764e0c092cac12f1R13

Reply:
I am using to get the content types on the ClientComponet file:

https://github.com/BuilderIO/builder/pull/1890/files#diff-374bce45b54565c4c1dc88f611c01ca689e1772670ebed8b024e73a3fd0a3e4d